js/miniQuery.js

```diff
@@ -0,0 +1,157 @@
+/*!
+ * minQuery
+ */
+"use strict"
+
+ var SweetSelector = {};
+
+ SweetSelector.select = function (cssSelector){
+  switch (cssSelector.charAt(0)) {
+  case '.':
+    return document.getElementsByClassName(cssSelector.slice(1));
+  case "#":
+    return document.getElementById(cssSelector.slice(1));
+  default:
+    return document.getElementsByTagName(cssSelector);
+  }
+ }
+
+var MiniQuery = function (cssSelector) {
+  if(typeof(cssSelector) == "function") {
+    return $.ready(cssSelector) }
+  else {
+    return SweetSelector.select(cssSelector)}
+};
+
+var $ = MiniQuery;
+
+MiniQuery.ready = function(thing){document.addEventListener('DOMContentLoaded', function(event){thing()})};
```

Comment:
why pass in the `event` variable to the callback function if `thing` does not accept it as a parameter?